js/components/controlBar.js

```diff
@@ -879,33 +876,38 @@ var ControlBar = createReactClass({
         alignment = CONSTANTS.TOOLTIP_ALIGNMENT.LEFT;
       }
       tooltipAlignments[collapsedControlBarItems[k].name] = alignment;
-      finalControlBarItems.push(controlItemTemplates[collapsedControlBarItems[k].name]);
+      var item = controlItemTemplates[collapsedControlBarItems[k].name];
+      if (this.props.equalSpacing) {
+        item = (
+          <div className="oo-flex-row" key={collapsedControlBarItems[k].name}>
+            {item}
+          </div>
+        );
+      }
+      finalControlBarItems.push(item);
     }
+
     return finalControlBarItems;
   },
 
   render: function() {
     var controlBarClass = ClassNames({
       'oo-control-bar': true,
-      'oo-control-bar-hidden': !this.props.controlBarVisible
+      'oo-control-bar-hidden': !this.props.controlBarVisible,
+      'oo-control-bar-video': !this.props.simpleControlBar
     });
 
     var controlBarItems = this.populateControlBar();
-    var controlBarStyle = {
-      height: this.props.skinConfig.controlBar.height
-    };
-
     return (
       <div
         className={controlBarClass}
-        style={controlBarStyle}
         onFocus={this.props.onFocus}
         onBlur={this.props.onBlur}
         onMouseUp={this.handleControlBarMouseUp}
         onTouchEnd={this.handleControlBarMouseUp}>
-        <ScrubberBar {...this.props} />
+        {!this.props.hideScrubberBar ? <ScrubberBar {...this.props} /> : null}
 
-        <div className="oo-control-bar-items-wrapper">
+        <div className="oo-control-bar-items-wrapper oo-flex-row-parent">
```

Comment:
Does this class need to be added conditionally for audio only mode?

Reply:
Nope, the control bar had flexbox styles on it before. I've moved them to oo-flex-row-parent.